specification/synopsis.markdown

```diff
@@ -26,25 +26,25 @@ cloud storage services such as Dropbox and MediaFire and use link-shortener
 services such as adf.ly to bring them a tiny bit of income. However, the
 launch of CurseForge changed the entire game: they offer mod file hosting
 services for no charges, recompense modders for their work, and provide a one-
-site modpack publishing services. Since then, the community started to shift
+site modpack publishing service. Since then, the community started to shift
 over everything to CurseForge, pushing Curse Inc. (later Twitch who acquired
-Curse Inc.) to the position of de facto standard of Modded Minecraft community.
+Curse Inc.) to the position of a de facto standard of Modded Minecraft community.
 
 Undeniably, CurseForge is now an indispensable part of the Modded Minecraft.
 Nevertheless, CurseForge is not the entire scene of this thriving community.
 There are mods that are not hosted on CurseForge, so CurseForge still has a
 list of non-CurseForge mods that are allowed to be included in a CurseForge
 modpack, alongside with an application mechanism for it. Mod reposting is still
 relevant; while we have taken steps to confront it, verifying whether a mod jar
-comes from authentic source is still a burden, especially for an absolute
-novice. Further, Modded Minecraft faces global audience, even though English is
+comes from the authentic source is still a burden, especially for an absolute
+novice. Further, Modded Minecraft faces the global audience, even though English is
 the conventional lingua franca in this community, users who are not proficient
 in English would still be annoying.
 
 Inspired by various package manager used by different Linux Distributions as
 well as the CKAN Project from Kerbel Space Program community, the Schema for
 Minecraft Mod Artifact Resource Tracking (SMMART) intends to set up a uniformed
-and platform-agnostic standard to record and exchange information about mod
+and platform-agnostic standard to record and exchange information about the mod
```

Comment:
is `the` here necessary?

Reply:
idk... ask 3t(

Reply:
"mod files" here does not refer to a *specific* mod, so no.